app/models/concerns/hashable.rb

```diff
@@ -1,11 +1,9 @@
 module Hashable
-  extend ActiveSupport::Concern
+  module_function
```

Comment:
TIL `module_function`. Nice.

Reply:
Nope, it applies to all the methods defined after it when used this way.